src/components/forms/Code/RememberCheckbox.vue

```diff
@@ -0,0 +1,26 @@
+<template>
```

Comment:
Do we really need to wrap atomic component with the same interface component?

Reply:
Artifact, it's not important now